cve_bin_tool/cli.py

```diff
@@ -73,18 +73,29 @@ def main(argv=None):
         + "\n\nPlease disclose issues responsibly!",
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
+
+    nvd_database_group = parser.add_argument_group(
+        "NVD", "Arguments related to NVD Datbase and Cache Configuration"
+    )
+    nvd_database_group.add_argument(
+        "-n",
+        "--nvd",
+        action="store",
+        choices=["api", "json"],
+        help="choose method for getting CVE lists from NVD",
+    )
+    nvd_database_group.add_argument(
+        "-u",
+        "--update",
+        action="store",
+        choices=["now", "daily", "never", "latest"],
+        help="update schedule for NVD database (default: daily)",
```

Comment:
Unrelated to this PR I guess but:

If you had `default="daily"` here, you could use `%(default)s`.

But right now most arguments don't have defaults set in the `add_argument`, I wonder why you use keep everything in the `defaults` dict instead, imo it's weird to have defaults set somewhere else. Why not set them when adding arguments, use them in help string and then just do something like `defaults = {key: parser.get_default(key) for key in vars(args)}` around line 237

Reply:
I believe the defaults dict was set up to integrate better with the config files, so the defaults can be loaded and over-written and I guess potentially correctly listed through --help but I don't think that actually happens.

Could be worth a small refactor to see if we can improve the display of defaults / loaded configs, but that's out of scope for this PR.  Feel free to open a new issue for it, though!